tools/Dockerfile

```
@@ -0,0 +1,216 @@
+# syntax=docker/dockerfile:1.1.3-experimental
+# This Dockfile contains separate targets for each testsys agent
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Shared build stage used to build Go binaries.
+ARG BUILDER_IMAGE
+FROM ${BUILDER_IMAGE} as build-go
+USER builder
+
+ARG GOARCH
+ARG GOOS=linux
+ARG GOROOT="/usr/libexec/go"
+ARG GOPROXY
+
+ENV PATH="${GOROOT}/bin:${PATH}"
+ENV GOPROXY="${GOPROXY}"
+
+ADD ./hashes /hashes
+COPY ./clarify.toml clarify.toml
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Shared build stage used to build Rust binaries.
+ARG BUILDER_IMAGE
+FROM ${BUILDER_IMAGE} as build-rust
+ARG ARCH
+
+USER root
+RUN mkdir -p /home/builder && \
+    chown -R builder:builder /home/builder
+USER builder
+WORKDIR /home/builder
+
+# We need these environment variables set for building the `openssl-sys` crate
+ENV PKG_CONFIG_PATH=/${ARCH}-bottlerocket-linux-musl/sys-root/usr/lib/pkgconfig
+ENV PKG_CONFIG_ALLOW_CROSS=1
+ENV CARGO_HOME=/home/builder/.cargo
+ENV OPENSSL_STATIC=true
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+FROM build-rust as boringtun
+
+ARG ARCH
+
+# TODO get licenses for boringtun
+RUN cargo install boringtun \
+    --target ${ARCH}-bottlerocket-linux-musl \
+    --root /home/builder
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+FROM build-go as eksctl-build
+
+USER root
+RUN mkdir -p /usr/share/licenses/eksctl && \
+    chown -R builder:builder /usr/share/licenses/eksctl
+
+ARG EKSCTL_VERSION=0.96.0
+ARG EKSCTL_SOURCE_URL="https://github.com/weaveworks/eksctl/archive/refs/tags/v${EKSCTL_VERSION}.tar.gz"
+
+ARG GOARCH
+ARG EKSCTL_BINARY_URL="https://github.com/weaveworks/eksctl/releases/download/v${EKSCTL_VERSION}/eksctl_Linux_${GOARCH}.tar.gz"
+
+USER builder
+WORKDIR /home/builder/
+RUN mkdir eksctl && curl -L ${EKSCTL_SOURCE_URL} \
+      -o eksctl_${EKSCTL_VERSION}.tar.gz && \
+    grep eksctl_${EKSCTL_VERSION}.tar.gz \
+      /hashes/eksctl | sha512sum --check - && \
+    tar -xf eksctl_${EKSCTL_VERSION}.tar.gz --strip-components 1 -C eksctl && \
+    rm eksctl_${EKSCTL_VERSION}.tar.gz
+
+WORKDIR /home/builder/eksctl/
+# TODO - remove this workaround for a bad go.sum hash
+# caused by https://github.com/blizzy78/varnamelen/issues/13#issuecomment-1079040934
+RUN \
+    # this sed is compatible with BSD and GNU versions of sed. it removes the version requirement in go.mod
+    sed -i.bak 's/go 1.18//' go.mod && \
+    # we need to delete and re-create the go.sum file to avoid a bad hash contained in it
+    rm go.sum && \
+    go clean -modcache && \
+    go mod tidy
```

```suggestion
RUN \
  sed -i 's,github.com/blizzy78/varnamelen v0.6.1 h1:kttPCLzXFa+0nt++Cw9fb7GrSSM4KkyIAoX/vXsbuqA=,github.com/blizzy78/varnamelen v0.6.1 h1:iYAU/3A6cpfRm2ZI0P/lece4jsc7GEbzsxTu+vBCChQ=,' go.sum ; \
  sed -i 's,github.com/blizzy78/varnamelen v0.6.1/go.mod h1:zy2Eic4qWqjrxa60jG34cfL0VXcSwzUrIx68eJPb4Q8=,github.com/blizzy78/varnamelen v0.6.1/go.mod h1:mGBHm+Uo4e8JnZEKHRoZgVEOQdSBdQfY/x+k4NAXBWA=,' go.sum
R
```
I would just fix the sums manually to keep things simple for now. I'm not sure if we'd rather make this a patch than a `sed` though.

---

This seems easier than a patch to me. Thank you I will try this.